frontend/js/components/Modal/Modal.component.js

```diff
@@ -0,0 +1,175 @@
+                  <label class="modal-label" for="dumpFilePath">Path of the Dump File</label>
+                  <input class="form-control load-db-input" aria-describedby="" type="text" name="dumpFilePath"
+                    id="dumpFilePath" autocomplete="off"
+                    onfocusout="validateInput(document.getElementById('dumpFilePath'), 'filePathError')" />
+                  <span class='formError' id='filePathError'></span>
+                </div>
+                <input type="text" style="display: none;">
+              </form>
+            </div>
+            <div class="modal-footer">
+              <input type="submit" disabled='disabled' value='Confirm' id='loadConnectButton' class='connectButton'
+                onclick='storeDumpFileValues(document.getElementById("loadDbType").value, document.getElementById("dumpFilePath").value)' />
+            </div>
+          </div>
+        </div>
+      </div>
+      
+
+      <div class="modal" id="connectToDbModal" tabindex="-1" role="dialog" aria-labelledby="exampleModalCenterTitle"
+  aria-hidden="true">
+  <div class="modal-dialog modal-dialog-centered" role="document">
+    <div class="modal-content">
+      <div class="modal-header content-center">
+        <h5 class="modal-title modal-bg" id="exampleModalLongTitle">Connect to Database</h5>
+        <i class="large material-icons close" data-dismiss="modal" onclick="clearModal()">cancel</i>
+      </div>
+      <div class="modal-body">
+        <div class="form-group">
+          <label for="dbType" class="">Database Type</label>
+          <select class="form-control db-select-input" id="dbType" name="dbType" onchange="toggleDbType()">
+            <option value="" style="display: none;"></option>
+            <option class="db-option" value="mysql">MySQL</option>
+            <option class="db-option" value="postgres">Postgres</option>
+            <option class='db-option' value='dynamodb'>dynamoDB</option>
+          </select>
+        </div>
+        <div id="sqlFields" style="display: none;">
+          <form id="connectForm">
+            <div class="form-group">
+              <label class="modal-label" for="dbHost">Database Host</label>
+              <input type="text" class="form-control db-input" aria-describedby="" name="dbHost" id="dbHost"
+                autocomplete="off" onfocusout="validateInput(document.getElementById('dbHost'), 'dbHostError')" />
+              <span class='formError' id='dbHostError'></span><br>
+            </div>
+            <div class="form-group">
+              <label class="modal-label" for="dbPort">Database Port</label>
+              <input class="form-control db-input" aria-describedby="" type="text" name="dbPort" id="dbPort"
+                autocomplete="off" onfocusout="validateInput(document.getElementById('dbPort'), 'dbPortError')" />
+              <span class='formError' id='dbPortError'></span><br>
+            </div>
+            <div class="form-group">
```

Comment:
modal component should not have the form details in the view
For componenets, along with the form items like the input, select will be separate components